ui/components/component-library/index.js

```diff
@@ -23,7 +23,7 @@ export { ButtonSecondary, BUTTON_SECONDARY_SIZES } from './button-secondary';
 export { FormTextField } from './form-text-field';
 export { HeaderBase } from './header-base';
 export { HelpText } from './help-text';
-export { Icon, ICON_NAMES, ICON_SIZES } from './icon';
+export { Icon, IconName, IconSize } from './icon';
```

Comment:
I also found the usage in `ui/components/component-library/icon/README.mdx`, if you can modify the import there would be great !

Reply:
Updated! https://github.com/MetaMask/metamask-extension/pull/18431/files#diff-fb571fb43371ebef2932c70a5e9f5f227892107e5143a9f6aae1de9f462edc2bR29